websites/P/PlanetXO/presence.ts

```diff
@@ -0,0 +1,161 @@
+		switch (document.location.href.split("/").slice(0, -1).join("/")) {
+			case "https://planetxo.uk": {
+				presenceData.details = "Viewing the map";
+				break;
+			}
+			case "https://planetxo.uk/xoradio": {
+				presenceData.details = "Viewing XO Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/xoradio",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel/planetxo": {
+				presenceData.details = "Viewing the PlanetXO Global Panel";
+				break;
+			}
+			case "https://planetxo.uk/panel/admin": {
+				presenceData.details =
+					"Viewing the PlanetXO System Administrator Panel";
+				break;
+			}
+			case "https://planetxo.uk/blastradio": {
+				presenceData.details = "Viewing Blast Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/blastradio",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/qtfm": {
+				presenceData.details = "Viewing QTFM";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/qtfm",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/lushradio": {
+				presenceData.details = "Viewing Lush Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/lushradio",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/everyradio": {
+				presenceData.details = "Viewing Every Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/everyradio",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel": {
+				presenceData.details = "Viewing PlanetXO Panel";
+				presenceData.buttons = [
+					{
+						label: "Join the Team?",
+						url: "https://planetxo.uk/vacancies.php",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel/xoradio": {
+				presenceData.details = "Viewing the Panel for XO Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/xoradio",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel/blastradio": {
+				presenceData.details = "Viewing the Panel for Blast Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/blastradio",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel/qtfm": {
+				presenceData.details = "Viewing the Panel for QTFM";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/qtfm",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel/lushradio": {
+				presenceData.details = "Viewing the Panel for Lush Radio";
+				presenceData.buttons = [
+					{
+						label: "Listen into the Tunes!",
+						url: "https://planetxo.uk/lushradio",
```

Comment:
Are these meant to be `/panel/<path>` or just `/<path>`? You can also use `document.location.href` to lead the user to the current path.

Reply:
IF THE USER is on /panel/[station] then it should be saying "Viewing the Panel for [station]".